src/igvfd/types/analysis_step.py

```diff
@@ -83,4 +83,5 @@ def _name(self, properties):
         'notSubmittable': True
     })
     def analysis_step_versions(self, request, analysis_step_versions):
-        return paths_filtered_by_status(request, analysis_step_versions)
+        analysis_step_versions = paths_filtered_by_status(request, analysis_step_versions)
+        return analysis_step_versions if analysis_step_versions else None
```

Comment:
This pattern can be simplified to `return analysis_step_versions or None` right? Same for all the others.